### PR TITLE
mark UpgradeSpec test as medium

### PR DESCRIPTION
**Pull request by remyhaemmerle-da**

### CODE REVIEW

**Review by paulbrauner-da — APPROVED** [review]

File: daml-lf/validation/BUILD.bazel [hunk 1]
```diff
@@ -51,13 +51,37 @@ da_scala_library(
     ],
 )
 
+updrage_test_file = "src/test/scala/com/digitalasset/daml/lf/validation/UpgradesSpec.scala"
```
Comment (samuel-williams-da) [inline daml-lf/validation/BUILD.bazel:54]: Typo? `upgrade_test_file`

**Review by samuel-williams-da — COMMENTED** [review]
This seems a little more involved than changing size, what else is happening here?

File: daml-lf/validation/BUILD.bazel (hunk 1 above)
Comment (remyhaemmerle-da) [reply daml-lf/validation/BUILD.bazel:54]: Good catch.

**Comment by remyhaemmerle-da** [comment]
At the end I chose `size = "medium"`, which corresponds to a 300s timeout. I think it should be enough

**Comment by samuel-williams-da** [comment]
Ah I see, thanks for clarifying

**Review by samuel-williams-da — APPROVED** [review]

**Pull request merged**